liquidctl/driver/nzxt_smart_device.py

```diff
@@ -163,29 +217,112 @@ def get_status(self, **kwargs):
+                    numValidRepliesReceived += 1 
+                    msg1101Reply = True
+                    continue
+                if msg2103Reply == False and msg[0] == 0x21 and msg[1] == 0x03:  # the correct reply must start with 0x21, 0x03
+                    ledStripCount = 0
+                    ledUnderglowCount = 0
+                    ledCableCombCount = 0
+                    ledAerFan120Count = 0
+                    ledAerFan140Count = 0
+                    for ledIter in range(15):
+                        ledCode = msg[0x0f + ledIter]
+                        if ledCode == 0x04:
+                            ledStripCount += 1
+                            ledName = '  LED Strip #{}'.format(ledStripCount)
+                        elif ledCode == 0x08:
+                            ledCableCombCount += 1
+                            ledName = ' Cable Comb #{}'.format(ledCableCombCount)
+                        elif ledCode == 0x0a:
+                            ledUnderglowCount += 1
+                            ledName = '  Underglow #{}'.format(ledUnderglowCount)
+                        elif ledCode == 0x0b:
+                            ledAerFan120Count += 1
+                            ledName = 'AER Fan 120 #{}'.format(ledAerFan120Count)
+                        elif ledCode == 0x0c:
+                            ledAerFan140Count += 1
+                            ledName = 'AER Fan 140 #{}'.format(ledAerFan140Count)
+                        if ledCode != 0:
+                            status.append(('    HUE2:', ledName, ''))
+                    numValidRepliesReceived += 1
+                    msg2103Reply = True
+                    continue
+                if msg6702Reply == False and msg[0] == 0x67 and msg[1] == 0x02:
+                    rpm_offset = 24
+                    duty_offset = 40
+                    noise_offset = 56
+                    for i, _ in enumerate(self._speed_channels):
+                        # LOGGER.debug('Iteration {}'.format(i))
+                        if ((msg[rpm_offset] != 0x0) and (msg[rpm_offset+1] != 0x0)): 
+                            status.append(('   Speed: Fan {}'.format(i+1), msg[rpm_offset+1] << 8 | msg[rpm_offset], 'rpm'))
+                            status.append(('    Duty: Fan {}'.format(i+1), msg[duty_offset+i], '%'))
+                        # else:
+                            # status.append(('Speed: Fan {}'.format(i+1), 'not', 'connected'))
+                        rpm_offset += 2
+                    status.append(('   Noise:', msg[noise_offset], 'dB'))
+                    numValidRepliesReceived += 1    
+                    msg6702Reply = True 
+                    continue            
+                if msg6704Reply == False and msg[0] == 0x67 and msg[1] == 0x04:
+                    #status.append((' Unknown:', 'status', 'pending'))
+                    numValidRepliesReceived += 1
+                    msg6704Reply = True
+                    continue
```

Comment:
If we don't care about this repply, remove all code and checks for it (if necessary, leave a comment stating what we might be able to get from parsing it).

Reply:
There may be something useful in this for a future update. Added a comment to that effect:

````
                if msg_6704_reply == False and msg[0] == 0x67 and msg[1] == 0x04:
                # PLACE HOLDER FOR FUTURE WORK
                # Interpretation of this reply is pending
                    #status.append((' Unknown:', 'status', 'pending'))
                    num_valid_replies_recvd += 1
                    msg_6704_reply = True
                    continue
````